mr/src/test/java/org/apache/iceberg/mr/mapreduce/TestIcebergInputFormat.java

```diff
@@ -142,7 +142,7 @@ public void testPartitionedTable() throws Exception {
                                 location.toString());
     List<Record> expectedRecords = RandomGenericData.generate(table.schema(), 1, 0L);
     expectedRecords.get(0).set(2, "2020-03-20");
-    DataFile dataFile = writeFile(table, Row.of("2020-03-20", 0), format, expectedRecords);
+    DataFile dataFile = writeFile(table, TestHelpers.Row.of("2020-03-20", 0), format, expectedRecords);
```

Comment:
Can you put these `TestHelpers` back to to how they were as these changes aren't in master (yes).